content/routing.md

```diff
@@ -32,7 +32,7 @@ meteor add kadira:flow-router
 
 *Snippet: Adding the Flow Router package*
 
-At the time of writing this guide, Flow Router is at version 2.x.
+At the time of writing this guide, [Flow Router](https://github.com/kadirahq/flow-router) is at version 2.x.
```

Comment:
Sound good to me. 

2. Here's the reason why I asked this: https://github.com/kadirahq/flow-router/issues/344
3. Not sure, just an idea I got.